app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split(",")
```

Comment:
I call function: `copy_file("cp old_file.txt new_file.txt")` and what i can get?